fit/core-reference/pom.xml

```diff
@@ -222,6 +222,30 @@ under the License.
       <artifactId>bcpkix-jdk15on</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Please remove all these added dependencies

Reply:
wow I did not add those depedencies, removing!